x/ccv/provider/keeper/keeper.go

```diff
@@ -316,12 +316,12 @@ func (k Keeper) DeleteUnbondingOp(ctx sdk.Context, id uint64) {
 }
 
 // This index allows retreiving UnbondingDelegationEntries by chainID and valsetUpdateID
-func (k Keeper) SetUnbondingOpIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64, IDs []uint64) {
+func (k Keeper) SetUnbondingOpIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64, index ccv.UnbondingOpsIndex) {
```

Comment:
Keep the signature `IDs []uint64` and put them into the proto struct in this function.

Reply:
`Done.`